src/compat/ge_408.ml

```diff
@@ -0,0 +1,28 @@
+(**********************************************************************
+ *                                                                    *
+ *                            OCamlFormat                             *
+ *                                                                    *
+ *  Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.  *
+ *                                                                    *
+ *  This source code is licensed under the MIT license found in the   *
+ *  LICENSE file in the root directory of this source tree.           *
+ *                                                                    *
+ **********************************************************************)
+
+let _ = Clflags.error_style := Some Misc.Error_style.Short
```

Comment:
Just FMI, `_` instead of `()` for any particular reason?

Reply:
none, will fix.